### DISCUSSION TITLE
¿Es posible utilizar este Add-in en Access en otros idiomas que no sea ingles?

### DISCUSSION THREAD

**Discussion by ApliNovaXXI** [joyfullservice/msaccess-vcs-addin#497] [Q&A]
He instalado este añadido (Add-in) en mi versión de Access 

Microsoft® Access® para Microsoft 365 MSO (versión 2402 compilación 16.0.17328.20124) de 32 bits 

Que esta en idioma Español (España).  Pero me da errores, pues creo que hay problemas con el idioma.

¿Que puedo hacer?

**Comment by joyfullservice** [comment]
Yes, I am aware of other users that are successfully using this add-in in other non-English languages. If you are encountering errors or technical issues with the functionality of the add-in in your language, please open an issue describing the technical details.

In the future I plan to add some translation support to the user interface so that users in other language environments can comfortably use the tool in their own native language.

**Reply by ApliNovaXXI** [reply]
He encontrado el problema.  Se trata que utilizo el complemento VBA vbWatchdog y este capturaba errores, que deberían ser controlados por MSAccessVCS.
Lo he solucionado de forma fácil, no dejando que se active vbWatchdog al cargar la aplicación, y utilizando posteriormente MSAccessVCS.

Gracias.

Si en algún momento se plantean adaptar esta utilidad al idioma español, estoy dispuesto a participar en medida de mis posibilidades.

**Reply by joyfullservice** [reply]
Thanks for letting me know what happened. I will keep you in mind when we get ready for adding translations. Thanks for your interest in helping!
